conans/client/embedded.py

```diff
@@ -0,0 +1,81 @@
+from conans.errors import ConanException
+from conans.paths import conan_expand_user
+from conans.client.paths import ConanPaths
+from conans.client.output import ConanOutput
+from conans import tools
+
+import yaml
+import os
+import sys
+
+class EmbeddedSettingsParser:
```

Comment:
remember to use modern python classes: ``class EmbeddedSettingsParser(object):``